sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj

```diff
@@ -7,13 +7,13 @@
     <PackageTags>Microsoft Azure Application Configuration</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-      - Enabled conditional requests.
-      - Added support for setting x-ms-client-request-id, x-ms-correlation-request-id, and correlation-context headers.
-      - Added SetReadOnly/ClearReadOnly methods.
-      - Enabled setting service version.
-      - Added support for Sync-Token headers.
-      - Updated authorization header format.
-      - Removed Update methods.
+      Refactored several classes based on feedback, including:
+      - Changed ConfigurationClient method names to align with client methods in other Azure SDK libraries.
+      - Make terminology consistent around IsReadOnly.
+      - Removed unneeded functionality.
+      In addition:      
+      - Fixed a bug causing incorrect request signing on retries.
+      - Made ConfigurationSetting serializable by System.Text.Json serializers.
```

Comment:
In KV release notes in the nupkg, we also added a URL to our changelog (the tag ID is stable and you can "guess" what it will be) for more details. Something to consider.